### ISSUE TITLE
[wxpython.py] Keyboard issues in DevTools on Windows

### ISSUE THREAD

**Issue by cztomczak** [cztomczak/cefpython#381]
When developer tools window is opened I can type "abcdef" characters, but other keys don't work:
- Enter
- arrows
- copy/paste commands

The backspace key works fine, so that's strange. All keys do work fine in main window.

### Temporary fix

A quick fix is to create "devtools.py" script that launches the devtools window in a separate process and can be called with `python devtools.py`. The hello_world.py example could be used as a base for that script.

### More debug info

- This issue doesn't occur on Mac, so looks like this is Windows only issue
- This issue doesn't occur in other examples, so it's a wxpython.py only issue.
- Opening devtools url using js window.open() doesn't help.
- Opening devtools url in MainFrame also doesn't resolve the issue.
- When pressing the Enter key many times it sometimes work. Sometimes a few key presses is required and sometimes dozens.
- A similar issue with Enter key not working occurs when wx.WANTS_CHARS style is not set for panel that embeds CEF browser. However when I loaded devtools url in MainFrame Enter key also didn't work even though wx.WANTS_CHARS was set. It seems that wxPython intercepts keyboard events for all application windows even those not created via wxPython eg. the devtools window created by CEF itself.
- Maybe no setting parent window handle for the CefWindowInfo structure could fix this. Source code in question here:
```
cpdef py_void ShowDevTools(self):
        cdef CefWindowInfo window_info
        IF UNAME_SYSNAME == "Windows":
            # On Windows with empty window_info structure the devtools
            # window doesn't appear.
            window_info.SetAsPopup(
                    <CefWindowHandle>self.GetOpenerWindowHandle(),  <------ parent window handle
                    PyToCefStringValue("DevTools"))
```

**Comment by cztomczak** [comment]
This issue doesn't occur on Mac, so looks like this is Windows only issue.

**Comment by cztomczak** [comment]
Fixed in commit da0a09f in the `dev` branch. Added [DevToolsHandler](https://github.com/cztomczak/cefpython/blob/dev/api/DevToolsHandler.md), the `devtools.py` script and updated `wxpython.py` example.
